tests/baselines/reference/inlayHintsInteractiveVariableTypes3.baseline

```diff
@@ -0,0 +1,128 @@
+// === Inlay Hints ===
+const lit1 = getTemplateLiteral1();
+          ^
+{
+  "text": "",
+  "displayParts": [
+    {
+      "text": ": "
+    },
+    {
+      "text": "`${"
+    },
+    {
+      "text": "string"
+    },
+    {
+      "text": "},${"
+    },
+    {
+      "text": ""
+    },
+    {
+      "text": "string"
+    },
+    {
+      "text": "}`"
+    }
+  ],
+  "position": 73,
+  "kind": "Type",
+  "whitespaceBefore": true
+}
+
+const lit2 = getTemplateLiteral2();
+          ^
+{
+  "text": "",
+  "displayParts": [
+    {
+      "text": ": "
+    },
+    {
+      "text": "`\\${${"
+    },
+    {
+      "text": "string"
+    },
+    {
+      "text": "},${"
+    },
+    {
+      "text": ""
+    },
+    {
+      "text": "string"
+    },
+    {
+      "text": "}`"
+    }
+  ],
+  "position": 175,
+  "kind": "Type",
+  "whitespaceBefore": true
+}
+
+const lit3 = getTemplateLiteral3();
+          ^
+{
+  "text": "",
+  "displayParts": [
+    {
+      "text": ": "
+    },
+    {
+      "text": "`start${"
+    },
+    {
+      "text": "string"
+    },
+    {
+      "text": "}\\${,$${"
+    },
+    {
+      "text": ""
```

Comment:
This looks wrong. I don't think we should generate empty display parts like these.

Reply:
Rpdated.